src/yoroi-wallets/cardano/transactionManager.ts

```diff
@@ -11,11 +11,11 @@ export const makeTransactionManager = async (storage: Storage) => {
     get transactions() {
       const {memos} = memosManager
       return Object.keys(memos).reduce(
-        (allTransactions, currentMemoAddress) => ({
+        (allTransactions, currentMemoTxAddress) => ({
```

Comment:
(result, current)
start with an empty {}

Reply:
iterate over that transactions instead of the memos

Reply:
don't bother considering performance optimizations. write code with the clearest intent.
`transactions -> transactions with memos`